cluster/addons/cluster-monitoring/standalone/heapster-controller.yaml

```diff
@@ -7,26 +7,23 @@
 apiVersion: v1
 kind: ReplicationController
 metadata:
-  name: heapster-v15
+  name: heapster-v1.0.0-beta1
   namespace: kube-system
   labels:
     k8s-app: heapster
-    version: v15
```

Comment:
Probably a dumb question, but without a unique label identifying the version, will updates to new versions work properly when we upgrade the master?